packages/components/app/styles/components/tooltip.scss

```diff
@@ -0,0 +1,80 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+//
+// TOOLTIP & TOOLTIP-BUTTON
+//
+
+@use "../mixins/focus-ring" as *;
+
+:root {
+  // Dimensions:
+  --token-tooltip-max-width: 280px;
+
+  // Focus highlight position
+  --token-tooltip-tab-focus-offset: -2px;
+
+  // Colors:
+  --token-tooltip-surface-primary: var(--token-color-palette-neutral-700);
+}
+
+.hds-tooltip-button {
+  @include hds-focus-ring-with-pseudo-element(
+    $top: var(--token-tooltip-tab-focus-offset),
+    $right: var(--token-tooltip-tab-focus-offset),
+    $bottom: var(--token-tooltip-tab-focus-offset),
+    $left: var(--token-tooltip-tab-focus-offset),
+  );
+  margin: 0;
+  padding: 0;
+  color: inherit;
+  background: transparent;
+  border: none;
+
+  > .flight-icon { display: block; }
```

Comment:
not a blocker, but consider having a `display: inline-flex` for the `--is-inline` variant, so you can remove this override

Reply:
Ok that's what I had before so I'll change it back...